crates/oxc_linter/src/rules/jest/padding_around_test_blocks.rs

```diff
@@ -0,0 +1,246 @@
+use itertools::Itertools;
+use oxc_ast::AstKind;
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::{GetSpan, Span};
+
+use crate::{
+    AstNode,
+    context::LintContext,
+    rule::Rule,
+    utils::{
+        JestGeneralFnKind, ParsedGeneralJestFnCall, collect_possible_jest_call_node,
+        parse_general_jest_fn_call,
+    },
+};
+
+fn padding_around_test_blocks_diagnostic(span: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn("Missing padding before test/it block")
+        .with_help("Make sure there is an empty new line before the test/it block")
```

Comment:
The return of `parse_general_jest_fn_call()` has a name property. Can you update the code that the jest fn call name is used